### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lynn915**

### CODE REVIEW

File: operations/workplace/people/performance-reviews-50/README.md [hunk 1]
```diff
@@ -54,3 +54,11 @@ We are incorporating a 90d New Hire Feedback Survey that will greatly mimic this
 * [Self Assessment Template](https://docs.google.com/forms/d/1ubNB9duV0YShDt7PYWDpjtHkSj6nFJl4HGCLqWpRjwQ/edit)
 * [Performance Evaluation - Manager's Template ](https://docs.google.com/document/d/1tTPTp8JC3pUp0kPo2AAUg6XETbURB_bU8f3EOpvKRr0/edit#)
```
Comment (justinegeffen) [inline operations/workplace/people/performance-reviews-50/README.md]:
```suggestion
* [Performance Evaluation - Manager's Template](https://docs.google.com/document/d/1tTPTp8JC3pUp0kPo2AAUg6XETbURB_bU8f3EOpvKRr0/edit#)
```

File: operations/workplace/people/performance-reviews-50/README.md [hunk 2]
```diff
@@ -54,3 +54,11 @@ We are incorporating a 90d New Hire Feedback Survey that will greatly mimic this
...
 
+## R&D Mid-Year Performance Review 2020 - Timeline & Forms
+
+* R&D Mid-Year Performance Review 2020 - Timeline
+* [Peer Review Survey](https://www.surveymonkey.com/r/3GJPBYH) Sent via Survey Monkey
```
Comment (justinegeffen) [inline operations/workplace/people/performance-reviews-50/README.md]:
```suggestion
* [Peer Review Survey](https://www.surveymonkey.com/r/3GJPBYH) - Sent via Survey Monkey
```

File: operations/workplace/people/performance-reviews-50/README.md [hunk 3]
```diff
@@ -54,3 +54,11 @@ We are incorporating a 90d New Hire Feedback Survey that will greatly mimic this
...
+* [Self-Assessment Template](https://drive.google.com/drive/u/0/folders/1qfgm6zfu5paP1j80CWHjEb3s0kHQi-rd) Sent via BambooHR
```
Comment (justinegeffen) [inline operations/workplace/people/performance-reviews-50/README.md]:
```suggestion
* [Self-Assessment Template](https://drive.google.com/drive/u/0/folders/1qfgm6zfu5paP1j80CWHjEb3s0kHQi-rd) - Sent via BambooHR
```

File: operations/workplace/people/performance-reviews-50/README.md [hunk 4]
```diff
@@ -54,3 +54,11 @@ We are incorporating a 90d New Hire Feedback Survey that will greatly mimic this
...
+* [Performance Evaluation - Manager's Template](https://drive.google.com/drive/u/0/folders/1qfgm6zfu5paP1j80CWHjEb3s0kHQi-rd)   Sent via BambooHR
```
Comment (justinegeffen) [inline operations/workplace/people/performance-reviews-50/README.md]:
```suggestion
* [Performance Evaluation - Manager's Template](https://drive.google.com/drive/u/0/folders/1qfgm6zfu5paP1j80CWHjEb3s0kHQi-rd) - Sent via BambooHR
```

**Review by natalie-hub — DISMISSED** [review]
Thank you @justinegeffen for editing!

**Review by jasonblais — DISMISSED** [review]

**Review by jasonblais — APPROVED** [review]

**Review by jasonblais — APPROVED** [review]

**Pull request merged**